setup.py

```diff
@@ -264,6 +264,7 @@
     'obspy.plugin.waveform.AH': [
         'isFormat = obspy.io.ah.core:_is_ah',
         'readFormat = obspy.io.ah.core:_read_ah',
+        'writeFormat = obspy.io.ah.core:_write_ah1'
```

Comment:
I don't know anything about AH versioning. I assume there are other AH versions as well? In that case - do you think it would make sense to have an `AH1` plugin?, e.g.

```python
'obspy.plugin.waveform.AH1': [
    'writeFormat = obspy.io.ah.core:_write_ah1'
]
```

The result is that writing AH files would be with `st.write("FILENAME", format="AH1")` instead of `format="AH"` like it currently is.